src/sass/_add-item-view.scss

```diff
@@ -0,0 +1,81 @@
+@use '_variables.scss' as *;
+
+.add-item-container {
+	position: relative;
+	align-items: center;
+	margin-bottom: 4em;
+
+	form {
+		.input {
+			display: flex;
+			flex-direction: column;
+			width: 75%;
+			margin: auto;
+		}
+
+		.input::after {
+			position: absolute;
+			top: 18%;
+			right: 4%;
+			content: url('/img/after.svg');
+		}
+
+		.item-urgency-container {
+			display: flex;
+			flex-direction: column;
+			padding: 30px 10px 0px 10px;
+
+			fieldset {
+				border: 0;
+				display: flex;
+				flex-direction: row;
+				flex-wrap: wrap;
+				justify-content: center;
+				padding: 30px;
+
+				input[type='radio'] {
+					display: none;
+					&:not(:disabled) ~ label {
+						cursor: pointer;
+					}
+				}
+
+				input[type='radio']:checked + label {
+					background: $purple;
+					color: $light-green;
+					box-shadow: 0px 0px 20px $yellow;
+				}
+
+				legend {
```

Comment:
To keep things super tidy, we can add the legend to the label property in `fonts-general-styles` & then we don't need it here and it will be easy to update if we want to change the colours.

```
//body fonts
label,
legend {
	font-family: $header-font;
	font-size: 2rem;
	text-align: center;
	color: $red-dark;
}
```

Reply:
Good call. I updated this in the latest changes.